### PR TITLE
Added support for Ox inv and added webhook check

### PR DESCRIPTION
**Pull request by lenzh**
Moved Webhook to Config.
Added Config.Inv for inventory support (qb ox)

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -1 +1,4 @@
 Config = Config or {}
+
+Config.Inv = "ox" -- qb(=lj) or ox [Inventory system]
+Config.webhook = "https://discord.com/api/webhooks/1098347143602913340/61nQVpgAZILNJyUNN3v6AP2yvtR8VLKcOjNVcfGYpKBfotrpqNy4uDxsQA-ERh4bqnFi" -- Add Discord webhook
```
Comment (MonkeyWhisper) [inline config.lua]: Move this to server sided please, so people can't dump it
And leave it as default qb, and remove your webhook

Thanks!

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

**Comment by lenzh** [comment]
Fixed on latest push

File: server/sv_main.lua [hunk 2]
```diff
@@ -24,45 +30,92 @@ RegisterNetEvent("ps-camera:CreatePhoto", function(url)
...
 
     local info = {
-        image = url,
+        ps_image = url,
         location = location
     }
-    player.Functions.AddItem("photo", 1, nil, info)
-    TriggerClientEvent('inventory:client:ItemBox', source, QBCore.Shared.Items['photo'], "add")
+    if not (Config.Inv == "qb" or Config.Inv == "ox") then return print('^1[Error] Your Config.Inv isnt set.. you probably had a typo\nYou have it set as= Config.Inv = "'.. Config.Inv .. '"') end
+
```
Comment (xFutte) [inline server/sv_main.lua]:
```suggestion
if not (Config.Inv == "qb" or Config.Inv == "ox") then 
    ConfigInvInvalid()
    return;
end
```

and then extract the print to a new local function.

**Review by xFutte — APPROVED** [review]

**Comment by xFutte** [comment]
Looks good to me. I'll let @MonkeyWhisper merge since he initially requested changes.

**Comment by xFutte** [comment]
image

Should be squashed before merging.

**Pull request merged**